### PR TITLE
refactor(app): audit of hardcoded slot names, OT-2 and Opentrons Flex

### PR DESCRIPTION
**Pull request by jerader**
# Overview

Audit of hardcoded slot names, Ot-2 and and Opentrons Flex:
1. removed unused i18n key that included a hardcoded slot name
2. removed a few i18n keys that are unused that included some hardcoded 'OT-2' strings
3. changed all instances of OT-3 in the ui to Opentrons Flex

Screen Shot 2023-05-04 at 1 42 39 PM

# Test Plan

- do a quick search throughout the desktop app and see that you dont' see Ot-3 and only see Opentrons Flex and that Ot-2 info remains the same. Look at stuff like: robot card, protocol card, protocol card overflow menu, send to ot-3 slideout, robot slideout, app settings, robot settings.
- quick search through ODD too

# Changelog

- removed a few unused i18n keys and changed all isntances of customer facing 'OT-3' strings to now say 'Opentrons Flex'
- fix affected tests
- changed the `getRobotModelByName` selector to return either OT-2 or Opentrons Flex.

# Review requests

Is it okay that i changed `getRobotModelByName` directly? Or should I leave that as is and convert from Ot-3 to Opentrons Flex downstream?

# Risk assessment

low

### CODE REVIEW

File: app/src/assets/localization/en/protocol_details.json [hunk 1]
```diff
@@ -30,7 +30,6 @@
   "start_setup": "Start setup",
   "run_protocol": "Run protocol",
   "show_in_folder": "Show in folder",
-  "start_setup": "Start setup",
```
Comment (jerader) [inline app/src/assets/localization/en/protocol_details.json]: this was a duplicate string from line 30

Comment (ewagoner) [reply app/src/assets/localization/en/protocol_details.json]: I see how it happened (two merges at nearly the same time) but I'm surprised the automated checks didn't catch that afterwards.

Comment (koji) [reply app/src/assets/localization/en/protocol_details.json]: this will be removed by my re-order pr.

Comment (jerader) [reply app/src/assets/localization/en/protocol_details.json]: cool i can remove this change, thanks Koji!

**Review by koji — APPROVED** [review]
lgtm! 
💪

**Pull request merged**